### PR TITLE
[B2BP-902] - Fix font of feature and howto components

### PR DESCRIPTION
**Pull request by salvatorediocaro**
#### List of Changes

Change font style of Feature and HowTo components

#### Motivation and Context

Modify style

#### How Has This Been Tested?

Run locally on localhost:3000 and localhost:6006

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Doesn't need any documentation update.

### CODE REVIEW

File: apps/nextjs-website/react-components/components/Feature/Subtitle.tsx [hunk 1]
```diff
@@ -11,24 +11,29 @@ const Subtitle = ({ item, theme }: FeatureStackItemProps) => {
...
-            href={item.link.href}
-            underline='none'
-            sx={{
-              fontWeight: 'bold',
-            }}
+          <Typography
+            component='span'
+            variant='body1'
+            color={theme === 'light' ? 'text.primary' : 'background.paper'}
+            fontSize={'18px'}
```
Comment (simosalsi) [inline apps/nextjs-website/react-components/components/Feature/Subtitle.tsx:26]:
```suggestion
```
I believe the `body1` variant already has a fontSize of 18 so no need to set it again.

File: apps/nextjs-website/react-components/components/HowTo/HowTo.tsx [hunk 2]
```diff
@@ -86,26 +90,34 @@ const HowTo = (props: HowToProps) => {
...
-            href={link.href}
-            target={link.target}
-            color={textColor}
-            underline='none'
-            fontWeight={600}
+          <Typography
+            component='span'
+            variant='body1'
+            color={theme === 'light' ? 'text.primary' : 'background.paper'}
+            fontSize={'18px'}
```
Comment (simosalsi) [inline apps/nextjs-website/react-components/components/HowTo/HowTo.tsx:97]:
```suggestion
```
I believe the `body1` variant already has a fontSize of 18 so no need to set it again.

File: apps/nextjs-website/react-components/components/HowTo/HowTo.tsx [hunk 3]
```diff
@@ -86,26 +90,34 @@ const HowTo = (props: HowToProps) => {
...
+            display={'contents'}
```
Comment (simosalsi) [inline apps/nextjs-website/react-components/components/HowTo/HowTo.tsx:98]: Is this necessary?

Not implying it isn't, just asking the reason why it's been used.

Comment (salvatorediocaro) [reply apps/nextjs-website/react-components/components/HowTo/HowTo.tsx:98]: only display={'contents'} was needed for link alignment, the rest was removed

**Pull request merged**